sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/RocksDB.scala

```diff
@@ -251,6 +251,11 @@ class RocksDB(
     }
   }
 
+  @GuardedBy("acquireLock")
+  private val rocksDBFileMapping: RocksDBFileMapping = new RocksDBFileMapping()
+
+  private val snapshots = new ConcurrentLinkedQueue[RocksDBSnapshot]()
+
```

Comment:
nit: at least one-line comment describing what each val is being used for will be nice for readers

Reply:
done